### PR TITLE
Update kestrel.md /path-4

### LINKED ISSUE #20177: cross link to Optional client certificates
> Some docs were added for this here:
> https://docs.microsoft.com/en-us/aspnet/core/security/authentication/certauth?view=aspnetcore-3.1#optional-client-certificates
> 
> Cross linking from the servers might help with discovery.

See https://github.com/dotnet/aspnetcore/issues/18064#issuecomment-707372095

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f22dcddb-0c25-be1a-9a02-9e71dce86bdb
* Version Independent ID: 3e0e12c2-e544-5655-e001-d2ec180b11a8
* Content: [Kestrel web server implementation in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.1)
* Content Source: [aspnetcore/fundamentals/servers/kestrel.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/servers/kestrel.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ec484002-18c1-6a0d-00de-820d56f038d3
* Version Independent ID: 7903ce53-2cbb-f083-7479-c8abc30211a0
* Content: [Host ASP.NET Core on Windows with IIS](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-3.1)
* Content Source: [aspnetcore/host-and-deploy/iis/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/iis/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #20177

### CODE REVIEW

File: aspnetcore/fundamentals/servers/kestrel.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ ASP.NET Core project templates use Kestrel by default. In *Program.cs*, the
 
 For more information on building the host, see the *Set up a host* and *Default builder settings* sections of <xref:fundamentals/host/generic-host#set-up-a-host>.
 
+## Optional client certificates
+
+See [Optional client certificates](xref:security/authentication/certauth#optional-client-certificates) on nformation for apps that must protect a subset of the app with a certificate.
```
Comment (guardrex) [inline aspnetcore/fundamentals/servers/kestrel.md:44]: If u don't go this way, the word "information" in the original needs a touch :point_up:.

```suggestion
For information on apps that must protect a subset of the app with a certificate, see [Optional client certificates](xref:security/authentication/certauth#optional-client-certificates).
```

File: aspnetcore/host-and-deploy/iis/index.md [hunk 2]
```diff
@@ -57,6 +57,10 @@ For getting started with hosting a website on Azure App Services, see our [deplo
 
 In general, we recommend using a pattern like [blue-green deployments](https://www.martinfowler.com/bliki/BlueGreenDeployment.html) for zero-downtime deployments. Features like Overlapped Recycle help, but don't guarantee that you can do a zero-downtime deployment. For more information, see [this GitHub issue](https://github.com/dotnet/aspnetcore/issues/10117).
 
+## Optional client certificates
+
+See [Optional client certificates](xref:security/authentication/certauth#optional-client-certificates) on nformation for apps that must protect a subset of the app with a certificate.
```
Comment (guardrex) [inline aspnetcore/host-and-deploy/iis/index.md:62]:
```suggestion
For information on apps that must protect a subset of the app with a certificate, see [Optional client certificates](xref:security/authentication/certauth#optional-client-certificates).
```

**Review by guardrex — APPROVED** [review]
Approve with suggestions.

File: aspnetcore/host-and-deploy/iis/index.md (hunk 2 above)
Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/iis/index.md]: Wow, much better wording! Thanks!!!

File: aspnetcore/fundamentals/servers/kestrel.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/servers/kestrel.md]:
```suggestion
See [Optional client certificates](xref:security/authentication/certauth#optional-client-certificates) on information for apps that must protect a subset of the app with a certificate. 
```

**Pull request merged**